appveyor.yml

```diff
@@ -72,10 +72,10 @@ install:
           else
             {$root = "C:\Miniconda-x64"}
           $env:path="$root;$root\Scripts;$root\Library\bin;$($env:path)"
+    - cmd: conda config --set show_channel_urls true
     - cmd: conda update --yes --quiet conda
     - cmd: set PYTHONUNBUFFERED=1
 
-    - cmd: conda config --set show_channel_urls yes
```

Comment:
Yeah, it's not a crisis as `defaults` is basically the only one. However, if we start wanting to use a newer version of jinja for instance, we might need to change that.